subcommand/consul-sidecar/command.go

```diff
@@ -187,9 +187,9 @@ func (c *Command) Help() string {
 	return c.help
 }
 
-const synopsis = "Connect lifecycle sidecar."
+const synopsis = "Connect consul sidecar."
```

Comment:
```suggestion
const synopsis = "Consul connect sidecar."
```

Reply:
Maybe it can be "Consul sidecar for Connect"?